packages/react-data-grid/src/__tests__/Row.spec.js

```diff
@@ -1,166 +0,0 @@
-import React from 'react';
-import { shallow } from 'enzyme';
-import TestUtils from 'react-dom/test-utils';
```

Comment:
I just nuked the couple tests that used `TestUtils.renderIntoDocument` out of laziness, and they didn't look very pertinent, let me know if you want them back.